_studio/shared/umc/codec/jpeg_dec/src/umc_mjpeg_mfx_decode_base.cpp

```diff
@@ -324,15 +324,15 @@ Status MJPEGVideoDecoderBaseMFX::_GetFrameInfo(const uint8_t* pBitStream, size_t
     int32_t   precision;
     JSS      sampling;
     JCOLOR   color;
-    CMemBuffInput in;
+    std::shared_ptr<CBaseStreamInput> in = std::make_shared<CMemBuffInput>();
     JERRCODE jerr;
 
     if (!m_IsInit)
         return UMC_ERR_NOT_INITIALIZED;
 
-    in.Open(pBitStream,nSize);
+    ((CMemBuffInput*)in.get())->Open(pBitStream,nSize);
```

Comment:
shared_pointer is a proxy of pointer, so think about it as a pointer, just use ->
```
in->Open(pBitStream,nSize);
```